client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,52 +1,55 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit, noop } from 'lodash';
 
-module.exports = React.createClass( {
-	displayName: 'SharingButtonsPreviewAction',
-
-	propTypes: {
-		active: React.PropTypes.bool,
-		position: React.PropTypes.oneOf( [
+export default class SharingButtonsPreviewAction extends Component {
+	static propTypes = {
+		active: PropTypes.bool,
+		position: PropTypes.oneOf( [
 			'top-left',
 			'top-right',
 			'bottom-left',
 			'bottom-right'
 		] ),
-		icon: React.PropTypes.string,
-		onClick: React.PropTypes.func
-	},
+		icon: PropTypes.string,
+		onClick: PropTypes.func
+	};
+
+	static defaultProps = {
+		active: true,
+		position: 'top-left',
+		onClick: noop
+	};
 
-	getDefaultProps: function() {
-		return {
-			active: true,
-			position: 'top-left',
-			onClick: function() {}
-		};
-	},
+	constructor( props ) {
```

Comment:
You don't need to define a `constructor` if all you're doing is calling `super`.